assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.stories.js

```diff
@@ -294,12 +294,12 @@ export const ZeroTileWithPlaceholder = Template.bind( {} );
 ZeroTileWithPlaceholder.storyName = 'ZeroTileWithPlaceholder';
 ZeroTileWithPlaceholder.args = {
 	configuredAudiences: [
-		'properties/12345/audiences/4', // Returning visitors
+		'properties/12345/audiences/1', // All Users
```

Comment:
Mocking the default audiences for the zero data state is more accessible than the Site Kit audiences.